canonical-web-and-design/jaas-dashboard
# Update Model / Controller icons

Fixes #296

## Done

Update Model / Controller icons

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify new icons in sidebar
